src/ui/components/pages/exportAccounts/passwordModal.tsx

```diff
@@ -0,0 +1,82 @@
+import * as styles from './passwordModal.styl';
+import cn from 'classnames';
+import * as React from 'react';
+import { Modal } from 'ui/components/ui/modal/Modal';
+import { Trans } from 'react-i18next';
+import { Input } from 'ui/components/ui/input';
+import { Error } from 'ui/components/ui/error';
+import { Button } from 'ui/components/ui/buttons/Button';
+
+interface Props {
+    onClose: () => void;
+    onSubmit: (password: string) => Promise<void>;
+}
+
+export function ExportAccountsPasswordModal({ onClose, onSubmit }: Props) {
+    const [password, setPassword] = React.useState('');
+    const [passwordError, setPasswordError] = React.useState(false);
+
+    return (
+        <Modal animation={Modal.ANIMATION.FLASH} showModal>
+            <div className="modal cover">
+                <form
+                    className="modal-form"
+                    onSubmit={async (event) => {
+                        event.preventDefault();
+                        setPasswordError(false);
+
+                        try {
+                            await onSubmit(password);
+                        } catch {
+                            setPasswordError(true);
+                        }
+                    }}
+                >
+                    <i className={cn(styles.lockIcon, 'lock-icon')} />
+
+                    <p className={cn('margin1', 'body1', 'disabled500')}>
+                        <Trans i18nKey="exportKeystore.passwordVerifyDesc" />
+                    </p>
+
+                    <div className="margin2 relative">
+                        <div className="basic500 tag1 input-title">
+                            <Trans i18nKey="exportKeystore.passwordLabel" />
+                        </div>
+
+                        <Input
+                            className="margin1"
```

Comment:
`autoFocus` didn't work out in that case, my only guess is because of the modal animation. So I did it using refs, not sure why it works that way, but not using `autoFocus`. I expected both of these ways to have an identical behavior. When I started to fix it, I was thinking of using `onEntered` callback from `CSSTransitionGroup`, but noticed that it works if `focus` called right after mounting.